style.css

```diff
@@ -0,0 +1,208 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  background: #3c3a39;
+  margin: 0;
+}
+
+.pure-nav-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-list {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.menu1 {
+  justify-content: end;
+  margin-top: 20px;
+  padding-right: 20px;
+}
+
+.menu {
+  width: 35px;
+  height: 3px;
+  color: #fff;
+  background-color: #acb7c3;
+  margin-top: 7px;
+  margin-right: 10px;
+}
+
+div a {
+  text-decoration: none;
+}
+
+div li {
+  list-style-type: none;
+}
+
+.important a {
+  color: #fff;
+  padding-left: 21px;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.mv-nav-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.mv-nav-list h4 {
+  color: #fff;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  margin-left: 10px;
+}
+
+.section-headline {
```

Comment:
- [ ] Your section-headline should have a display of flex, and the contents should be aligned to the center with margins
Your contents seems to align to the left which is different from the actual design
your design
![ay](https://user-images.githubusercontent.com/53356820/143458378-7ba23622-44a0-41c5-96d5-9e1ddad1d1ed.png)

Figma design
![ay1](https://user-images.githubusercontent.com/53356820/143458523-1ded4182-ce43-4b95-a665-75e575db3be2.png)

Reply:
Thank you. My 'section-headline' now has a display of flex.

![image](https://user-images.githubusercontent.com/91897740/143504254-548482ec-011b-45f3-b4da-914beb19cc4a.png)

Reply:
I have also modified the font to display in the same way as the Figma Template.